### PR TITLE
[WIP] Prep release

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

**Comment by jensengar** [comment]
I'd like to get a candidate out we can test headsets against. Let's not merge this until we've done tests.

File: test/unit/media/headset.test.ts [hunk 1]
```diff
@@ -167,6 +167,31 @@ describe('SdkHeadset', () => {
...
+      sdkHeadset.setRinging = jest.fn();
+
+      sdk.emit('pendingSession', { autoAnswer: false, conversationId: '15215asdf'} as any);
+      expect(sdkHeadset.setRinging).toHaveBeenCalled();
+    });
+
+    it('should not set ringing if autoanswer', () => {
+      sdkHeadset.setRinging = jest.fn();
+
+      sdk.emit('pendingSession', { autoAnswer: true, conversationId: '15215asdf'} as any);
```
Comment (davidmorton-gen) [inline test/unit/media/headset.test.ts:191]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9553941)
```suggestion
      sdk.emit('pendingSession', { autoAnswer: true, conversationId: '15215asdf'} as unknown);
```

File: test/unit/media/headset.test.ts [hunk 2]
```diff
@@ -167,6 +167,31 @@ describe('SdkHeadset', () => {
       expect(setHoldSpy).toHaveBeenCalledWith('123', false);
     });
   });
+
+  describe('listenForSessionEvents', () => {
+    it('should end currentCall on sessionEnded', () => {
+      sdkHeadset.endCurrentCall = jest.fn();
+
+      const conversationId = 'myConvoId';
+      sdk.emit('sessionEnded', { conversationId } as any, null);
```
Comment (davidmorton-gen) [inline test/unit/media/headset.test.ts:176]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9553941)
```suggestion
      sdk.emit('sessionEnded', { conversationId } as unknown, null);
```

File: test/unit/media/headset.test.ts [hunk 3]
```diff
@@ -167,6 +167,31 @@ describe('SdkHeadset', () => {
...
+
+      expect(sdkHeadset.endCurrentCall).toHaveBeenCalledWith(conversationId);
+    });
+
+    it('should set ringing if not autoanswer', () => {
+      sdkHeadset.setRinging = jest.fn();
+
+      sdk.emit('pendingSession', { autoAnswer: false, conversationId: '15215asdf'} as any);
```
Comment (davidmorton-gen) [inline test/unit/media/headset.test.ts:184]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9553941)
```suggestion
      sdk.emit('pendingSession', { autoAnswer: false, conversationId: '15215asdf'} as unknown);
```

File: package.json [hunk 4]
```diff
@@ -60,11 +60,11 @@
     "browserama": "^3.1.1",
     "core-js": "^3.7.0",
     "genesys-cloud-client-logger": "^4.1.0",
-    "genesys-cloud-streaming-client": "^13.4.1",
+    "genesys-cloud-streaming-client": "^14.0.0",
```
Comment (djhouseknecht) [inline package.json:63]: Need to bump to 14.0.1

**Pull request closed without merging**